public/app/features/alerting/unified/api/grafana.ts

```diff
@@ -76,60 +71,6 @@ export async function fetchContactPointsState(alertManagerSourceName: string): P
     );
     return contactPointsStateDtoToModel(response.data);
   } catch (error) {
-    if (isFetchError(error)) {
-      if (!isContactPointsStateNotAvailable(error)) {
-        return contactPointsStateDtoToModel([]);
-      } else {
-        throw error;
-      }
-    } else {
-      throw error;
-    }
+    return contactPointsStateDtoToModel([]);
```

Comment:
Good point @VikaCep. Exactly, I realized we don't need to distinguish between request error and empty response. We only show these notifications state in case we have a non-empty list in response.